<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetConvOptionsColor</class>
 <widget class="QWidget" name="WidgetConvOptionsColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>75</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Bits per point: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Red:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxRedbits">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Green:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxGreenBits">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Blue:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBlueBits">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Colors order:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxColorsOrder"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxRedbits</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WidgetConvOptionsColor</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxGreenBits</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WidgetConvOptionsColor</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBlueBits</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WidgetConvOptionsColor</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxColorsOrder</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>WidgetConvOptionsColor</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePreview()</slot>
 </slots>
</ui>
